tests/queries/0_stateless/02416_rename_database_rbac.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
```

Comment:
This may be unneeded, because I copy-pasted the test

Reply:
`no-parallel` tag can be avoided by using unique database and user name, it's quite easy to do in `.sh` test:
```
${CLICKHOUSE_CLIENT} --multiline --multiquery -q """
DROP DATABASE IF EXISTS database_02416_${CLICKHOUSE_DATABASE};
CREATE DATABASE database_02416__${CLICKHOUSE_DATABASE};
DROP USER IF EXISTS user_test_02416__${CLICKHOUSE_DATABASE};
CREATE USER user_test_02416__${CLICKHOUSE_DATABASE} IDENTIFIED WITH plaintext_password BY 'user_test_02416';

```